corehq/apps/locations/bulk_management.py

```diff
@@ -834,6 +835,8 @@ def update_children(parent_stub):
         child_stubs = location_stubs_by_parent_code[parent_code]
 
         for child_stub in child_stubs:
+            if excel_importer:
+                excel_importer.add_progress()
```

Comment:
When `excel_importer` is instantiated, [self.total_rows is set](https://github.com/dimagi/commcare-hq/blob/70630f7ace5299449e40ffb0cf9b928e2bd1cd98/corehq/util/workbook_json/excel_importer.py#L58-L58) to the number of location types + the number of locations. `add_progress` should increment progress by 1 location.

BUT I did forget to call `add_progress()` for location types. Pushing a commit for that now.

I'll ping when it's tested.